### PR TITLE
Auto assign list update

### PR DESCRIPTION
**Pull request by greymistcube**
PR reviewer auto assign list updated.

### CODE REVIEW

File: .github/auto_assign.yml [hunk 1]
```diff
@@ -7,5 +7,6 @@ reviewers:
 - limebell
 - riemannulus
 - moreal
```
Comment (dahlia) [inline .github/auto_assign.yml:9]: Could you remove @moreal?

```suggestion
```

**Review by moreal — DISMISSED** [review]
👋🏻

**Review by dahlia — CHANGES_REQUESTED** [review]

File: .github/auto_assign.yml (hunk 1 above)
Comment (greymistcube) [reply .github/auto_assign.yml]: Done.

**Review by longfin — DISMISSED** [review]

**Review by moreal — APPROVED** [review]
😿 👋🏻

**Review by dahlia — DISMISSED** [review]

**Review by dahlia — APPROVED** [review]

**Review by longfin — APPROVED** [review]

**Pull request merged**